.gitignore

```diff
@@ -1,5 +1,6 @@
 /node_modules
 .idea
+.*.swp
```

Comment:
This belongs in your personal global `.gitignore`, not in our project specific one. Please remove.